server/api/dataset.py

```diff
@@ -128,6 +129,10 @@ def ingestDataset(self, params):
                 'Invalid upload folder ID.', 'uploadFolderId')
 
         name = params['name'].strip()
+        owner = params['owner'].strip()
+        if not owner:
```

Comment:
Since I made `owner` a required parameter, it was an oversight that I didn't include it in the initial call to `requireParams`. Nevertheless, it's explicitly checked to be nonempty again after `strip`, since `requireParams` doesn't inspect the content.